src/net.cpp

```diff
@@ -2138,7 +2139,10 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
             }
         }
 
-        if (InactivityCheck(*pnode)) pnode->fDisconnect = true;
+        if (InactivityCheck(*pnode)) {
+            LogPrint(BCLog::NET, "disconnecting due to inactivity peer=%d\n", pnode->GetId());
```

Comment:
`InactivityCheck()` already logs a reason for any case where it returns true, eg:

```
2023-09-26T07:21:14.013289Z [net] socket receive timeout: 1201s peer=745   <==== this one
2023-09-26T07:21:14.014945Z [net] disconnecting peer=745
2023-09-26T07:21:14.016616Z [net] Cleared nodestate for peer=745

2023-09-26T08:33:15.016157Z [net] version handshake timeout peer=9862    <=== and this one
2023-09-26T08:33:15.016539Z [net] disconnecting peer=9862
2023-09-26T08:33:15.017321Z [net] Cleared nodestate for peer=9862
```

Does an extra log entry here actually help? Would adding the word "disconnecting" to the log entries in `InactivityCheck()` be better?

Reply:
Done, and also made sure the keyword `disconnecting` appears in all cases where we disconnect.